src/app/functions/server/applyCoupon.ts

```diff
@@ -67,8 +67,7 @@ export async function checkCurrentCouponCode(
   const currentCouponCode = process.env.CURRENT_COUPON_CODE_ID;
   if (!currentCouponCode) {
     logger.error(
-      "fxa_check_coupon_failed",
-      "Coupon code ID is not set. Please set the env var: CURRENT_COUPON_CODE_ID",
+      "fxa_check_coupon_failed: Coupon code ID is not set. Please set the env var: CURRENT_COUPON_CODE_ID",
```

Comment:
Do you know what this looks like in the logging console? Is it possible to still look these up with `jsonPayload.message='fxa_check_coupon_failed'` or does the second argument need to be an object?

Reply:
Yea let me rework this

Reply:
I think the second arg needs to be an obj, it wasn't showing. That's why I appended it to the first arg